lib/gollum/views/helpers.rb

```diff
@@ -61,7 +61,9 @@ def clean_url(*url)
         url.compact!
         return nil if url.empty?
 
-        ::File.join(*url).gsub(%r{/{2,}}, '/')
+        _url = ::File.join(*url)
+        _url = _url.gsub(%r{/{2,}}, '/')
```

Comment:
The following would be faster:

```ruby
_url.gsub!(%r{/{2,}}, '/')
_url.gsub!(%r{\?}, '%3F')
_url
```

Reply:
Done! https://github.com/ViChyavIn/gollum/commit/7643cd3b421bc0b4b19a27a5a583c76ae32c1b64